tests/release/conftest.py

```diff
@@ -132,29 +215,3 @@ def wrapper(*args: Any, **kwargs: Any) -> pd.DataFrame:
         return wrapper
 
     return decorator
-
-
-def parse_dataset_params(
```

Comment:
I replaced this with a fixture called dataset_params but the logic is all the same